### PR TITLE
tests: run the master tests against the staging server

### PR DESCRIPTION
**Pull request by elopio**
I added TEST_STORE=staging to the travis dashboard. This variable will only
be used by branches from the snapcore organization. So when travis runs the
tests after a branch has landed, it will use the staging server. All the
pull requests coming from personal branches will run the tests against the
fake servers.

LP: [#1668717](https://bugs.launchpad.net/snapcraft/+bug/1668717)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,14 +9,10 @@ virtualenv:
   system_site_packages: true
 
 env:
-  global:
-  # Encrypted test user password. Generated with the travis gem:
-  #     travis encrypt -r ubuntu-core/snapcraft TEST_USER_PASSWORD=$password
-  - secure: "gqtqTji8cie0Q2O+sRhE4MbTXGI0qTq8yPgRGFd9XlT/lB/EttFQKu72qycr/jyrvt809wjWM13QVqa5/71SoJd+Xzrmr1/leevx9Z/Wnv+IYkRAuGHW7iIDQb7MhQvpq3tw8hbGJzGxw03cUmjKJ89AAlGbwaURMat47lPsRXus8R7pl9S6r5owhBbmrQNaP9io0oPQDOAUf4pmJma1FTHAjXg0EdUwdXFUWToj15c7UJtB/MQNNTfjlwGA+/sPDqgthUEAzXmvUfXAZWnjQFZmq4ebvBIJEOQEPdLCXWGYdN2DAL7zp7WthrwFfgFFZb579rOBh0ETIMebUgBLoVSiPcn/bfzdYHcYKGf7lTJpoug5QENl+kZcuVyK7GUjf8O9tamhkYeMtUOy5Ubrcnv+Lfy9NsDPhKY05n+7tzzUVB1dePTrMHPuRZLl4OKku1AUN/S3A2xMrLO8vsWPVxcfxeb+4Y5ikYiHHpOozJHHDdPmj5raRIf3IH87W2PX0nJhg+gEgNHV1v3HBoyeqOPl4hl6/Fb9sCS/JAbbfcixkC54MHHI+opNSgZRvY0RORGHmuhHRGvfMxnwHmeOD51oV+SRGJS6A7qUq6GlBIy3/YlAY3LqqkWrHwm4EYttd4yM1FZ5s9pVnoSSJkgQ5vYK7A8a9AgZJlCcZnbzXtA="
```
Comment (elopio) [inline .travis.yml]: I also removed TEST_USER_PASSWORD from the yaml and added it on the dashboard, to be able to change it easily.

**Comment by kyrofa** [comment]
> So when travis runs the tests after a branch has landed, it will use the staging server. All the pull requests coming from personal branches will run the tests against the fake servers.

This is because it uses the key, I assume? I figure the committer will get an email after their PR merges and the tests fail, but will anyone else?

**Comment by elopio** [comment]
Yes, it uses the key. And I get travis emails when it fails, I'm not totally sure why. @kyrofa don't you get emails when master is red?

**Comment by kyrofa** [comment]
> @kyrofa don't you get emails when master is red?

Only when I was a member of the party of that commit to master (e.g. either I made the PR that got merged, or merged it).

**Comment by elopio** [comment]
we can set the email for notifications: https://docs.travis-ci.com/user/notifications/#Configuring-email-notifications

Not sure what we should put there. Probably it's better to send the notification to snapcraft-team in rocket chat. That shouldn't be too hard, I assume.

**Pull request merged**